### PR TITLE
Alal/docs

### PR DESCRIPTION
**Pull request by avantikalal**
Updated sphinx docs

### CODE REVIEW

File: docs/pretrained_models.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+
+The training data for these models was generated from BAM files using `this script <https://github.com/zchiang/atacworks_analysis/blob/master/preprocessing/atac_bam2bw.sh>`_. 
+
+Briefly, each ATAC-seq read was converted to a single genomic position corresponding to the first base pair of the read. Reads aligning to the + strand were offset by +4 bp, while reads aligning to the - strand were offset by -5 bp. Each cut site location was extended by 100 bp in either direction. The bedtools genomecov function was used to convert the list of locations into a coverage track. To call peaks from clean and noisy signal tracks, MACS2 subcommands bdgcmp and bdgpeakcall were run with the ppois parameter and a -log10(p-value) cutoff of 3. BED files with equal coverage over all chromosomes were provided as a control input track.
+
+Before using these models, please note that they should only be used on data that is processed exactly as described above. If your bigWig files and/or peak calls are generated using a different method, these models may give unreliable results, and it would be preferable to train your own model as described in Tutorial 1 (link to tutorial 1).
```
Comment (ntadimeti) [inline docs/pretrained_models.rst:11]: Could you link to tutorial 1 at the end of this paragraph

Comment (avantikalal) [reply docs/pretrained_models.rst:11]: Not sure what to link to. Jupyter notebook? Sphinx page?

Comment (ntadimeti) [reply docs/pretrained_models.rst:11]: Let's link to the md file for this PR. That way, this PR will be complete on it's own. Once I merge the jupyter notebooks, we can open another PR to modify the tutorial md files to point to the jupyter notebook.

Comment (ntadimeti) [reply docs/pretrained_models.rst:11]: @avantikalal let's merge this without the link. I will open another PR to look into this and fix the readme gif and couple other links.

**Comment by avantikalal** [comment]
Going to make one minor change first.

**Comment by ntadimeti** [comment]
Sure! feel free to merge whenever you'd like.

**Pull request merged**